pkgs/development/haskell-modules/configuration-ghc-8.2.x.nix

```diff
@@ -5,7 +5,7 @@ with haskellLib;
 self: super: {
 
   # Suitable LLVM version.
-  llvmPackages = pkgs.llvmPackages_39;
+  llvmPackages = pkgs.llvmPackages;
```

Comment:
This change is fine. The oldest LLVM version we actually need is `llvmPackages_5` for ghc-8.4.4.

ghc-8.2.2 is a binary downloaded from haskell.org that we use for bootstrapping our ghc-8.4.4 and 8.6.5 compilers. We don't need LLVM for that.

Reply:
thx for reviewing, I will leave it then as is.